src/components/form/web-forms-settings-confirmation.vue

```diff
@@ -53,7 +53,8 @@ except according to the terms contained in the LICENSE file.
         {{ $t('enketoConfirmation.description') }}
       </p>
       <div class="modal-actions">
-        <button class="btn btn-link" type="button" @click="$emit('hide')">
+        <button class="btn btn-link" type="button"
+          :aria-disabled="form.awaitingResponse" @click="$emit('hide')">
```

Comment:
This change isn't really related to the rest of this PR. I just noticed while I was looking at prior art that this button didn't have an `aria-disabled` attribute.